idaes/models/properties/examples/methanol_ideal_VLE.py

```diff
@@ -22,17 +22,18 @@
 
```

Comment:
As part of the broader question of where this example belongs, I wonder if it would be better to keep the property modules with the flowsheet so that it is easier for users to find them,